vitrina/datasets/managers.py

```diff
@@ -1,6 +1,7 @@
 from django.db import models
+from django.db.models import Q
 
 
 class PublicDatasetManager(models.Manager):
     def get_queryset(self):
-        return super().get_queryset()
+        return super().get_queryset().filter(~Q(is_public=False))
```

Comment:
```suggestion
        return super().get_queryset().filter(is_public=True)
```

I think there is no need to use `~Q` here.

Also, this filter is not complete, for example it does not filter out deleted datasets. Please make sure, that this filter does exactly the same thing as in current data.gov.lt version.